lib/build/script_compactor.dart

```diff
@@ -81,11 +81,11 @@ class ScriptCompactor {
 
   Future<Asset> run() {
     var crawler = new ImportCrawler(transform, primaryInput, logger);
-    return crawler.crawlImports().then((imports) {
+    return crawler.crawlImports().then((imports) async {
```

Comment:
This async/await stuff shouldn't be necessary afaik, if there is a strong mode error that is probably a bug.